README.md

```diff
@@ -340,16 +491,55 @@ After writing to the captureOuput.pcap file you can read the file with the follo
 
 `tshark -r captureOutput.pcap`
 
-TShark offers a plethora of options to help specify any search vectors. A few examples of some options would be as follows.
+With having some basics covered, we may now get into some scripts written within different programming languages. Some languages that will be covered are BASH, Python, and Ruby.  
+
+### BASH
+
+```
+function save_ping() {
+    tshark -w ping.pcap -f "host 8.8.8.8" -c 1
+    ping 8.8.8.8 -c 1
+}
+save_ping
+```
+The script above generates traffic with netcat, ping, hping, etc. and saves it with tshark.
```

Comment:
I assume this description is true for the Python script below as well? If so, maybe copy it or rewrite down there too? If that one is a completely different script, maybe add a description.